assets/js/modules/reader-revenue-manager/components/setup/SetupForm.js

```diff
@@ -105,7 +119,7 @@ export default function SetupForm( { onCompleteSetup } ) {
 				<PublicationSelect />
 			</div>
 			<PublicationOnboardingStateNotice />
-			<Link external href={ serviceURL }>
+			<Link external href={ serviceURL } onClick={ resetPublications }>
```

Comment:
Why do we reset publications when navigating to publisher center? I'd think we'd want to trigger a reset after SK is refocused instead, or use another action to "check again" or similar. If a user creates a new publication, they'd need to click this again for a new publication to show up in SK, no?

Reply:
We actually do not reset here. We only set a `core/forms` form data here, which in turn runs the reset on refocus in the `SetupMain` component.

Reply:
Oh great, thanks for clarifying. It's a bit confusing based on the naming then, I thought it was the `resetPublications` action. Better to not use the same naming for different things. As an event handler, I find it's best to use the convention like `handleLinkClick` or something like that.